### DISCUSSION TITLE
[categorie_corrigee] Cas 8 : limite de distance 50km

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#12] [Q&A] [answered]
Petite question mineure:
![image](https://user-images.githubusercontent.com/46813298/103225088-762bc980-4929-11eb-8821-0bbdc699fd9f.png)
Le cas de la distance exacte de 50km n'est pas pris en compte.

Dans l'implémentation, j'ai considéré supérieure à 50km et inférieur **ou égal** à 50km. Mais je peux changer si besoin.

**Comment by JMPivette** [comment]
inférieur **ou égal** à 50 km semble cohérent. C'est ce qui est utilisé pour les cas 6 et 11.

**Comment by SteffMangin** [comment, marked as answer]
oui
